Error executing job with overrides: ['env=lunarlander_v2', 'epsilon=glie', 'train_episodes=500000']
An error occurred during Hydra's exception formatting:
AssertionError()
Traceback (most recent call last):
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/_internal/utils.py", line 254, in run_and_report
    assert mdl is not None
AssertionError
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/scratch/work/zhangy50/Aalto_RL/ex3/train.py", line 152, in <module>
    main()
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/main.py", line 90, in decorated_main
    _run_hydra(
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/_internal/utils.py", line 389, in _run_hydra
    _run_app(
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/_internal/utils.py", line 452, in _run_app
    run_and_report(
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/_internal/utils.py", line 296, in run_and_report
    raise ex
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/_internal/utils.py", line 213, in run_and_report
    return func()
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/_internal/utils.py", line 453, in <lambda>
    lambda: hydra.run(
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/_internal/hydra.py", line 132, in run
    _ = ret.return_value
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/core/utils.py", line 260, in return_value
    raise self._return_value
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/hydra/core/utils.py", line 186, in run_job
    ret.return_value = task_function(task_cfg)
  File "/scratch/work/zhangy50/Aalto_RL/ex3/train.py", line 94, in main
    env = gym.make(cfg.env_name, render_mode='rgb_array' if cfg.save_video else None)
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/gym/envs/registration.py", line 619, in make
    env_creator = load(spec_.entry_point)
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/gym/envs/registration.py", line 62, in load
    mod = importlib.import_module(mod_name)
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/gym/envs/box2d/__init__.py", line 1, in <module>
    from gym.envs.box2d.bipedal_walker import BipedalWalker, BipedalWalkerHardcore
  File "/scratch/work/zhangy50/.conda_envs/rl/lib/python3.10/site-packages/gym/envs/box2d/bipedal_walker.py", line 25, in <module>
    raise DependencyNotInstalled("box2D is not installed, run `pip install gym[box2d]`")
gym.error.DependencyNotInstalled: box2D is not installed, run `pip install gym[box2d]`